client/src/components/squids/SquidList.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import { useSquidList } from "../hooks/useSquidList";
+import { SquidListTile } from "./SquidListTile";
+
+import "../../style/squids/squidList.pcss";
+
+export const SquidList = () => {
+  const squidListQuery = useSquidList();
```

Comment:
I really like this variable naming -- usually I deconstruct right away, so 

```
const {data, isLoading, isError, error} = useSquidList();
const squids = data || [];
```

but I think your approach would be more readable than mine in a component with multiple queries! (I would then `{ data: squidData, .... } = useSquidList()`, but still)